### PR TITLE
fix: input types loose out on provided graphQL documentation

### LINKED ISSUE #1625: bug(config): input types loose out on provided graphQL documentation
Documentation isn't copied into the blueprint for input types from configurations. For eg:

```graphql
"""
Some Documentation
"""
input Foo {
  id: Int!
}
```

While using a configuration with an input type like the above, documentation isn't copied in the final schema. 

**Technical Requirements**
- Copy the documentation into the blueprint
- Ensure integration tests are added

### PR DESCRIPTION
**Pull request by emekaokoli19**
**Summary:**  
Add descriptions to input object types.

**Issue Reference(s):**  
Fixes #1625

**Build & Testing:**

- [x] I ran `cargo test` successfully.
- [x] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

**Checklist:**

- [x] I have added relevant unit & integration tests.
- [ ] I have updated the [documentation] accordingly.
- [x] I have performed a self-review of my code.
- [ ] PR follows the naming convention of `<type>(<optional scope>): <title>`

[documentation]: https://github.com/tailcallhq/tailcallhq.github.io/tree/develop/docs

/claim #1625

### CODE REVIEW

**Comment by emekaokoli19** [comment]
@tusharmath I traced through the codebase and found the problem, I also fixed the field documentation on input objects.

File: tests/core/snapshots/test-input-documentation.md_client.snap [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+type Query {
+  foo: String
+  postFromFoo(id: Int!): Post
+}
+
+scalar Url
+
+schema {
```
Comment (tusharmath) [inline tests/core/snapshots/test-input-documentation.md_client.snap:36]: Documentation is still missing here.

Comment (emekaokoli19) [reply tests/core/snapshots/test-input-documentation.md_client.snap:45]: @tusharmath I am sorry about that, the documentation wasn't passed to schema. I have pushed a fix for it.

**Review by tusharmath — APPROVED** [review]

**Comment by tusharmath** [comment]
Wired, why did this not get merged?

**Pull request merged**